gprofiler/client.py

```diff
@@ -52,19 +55,26 @@ def get_base_url(self, api_version: str = None) -> str:
         version = api_version if api_version is not None else self._version
         return "{}/{}/{}".format(self._host.rstrip("/"), self.BASE_PATH, version)
 
+    def _get_query_params(self) -> List[Tuple[str, str]]:
+        return [("key", self._key), ("service", self._service), ("hostname", self._hostname)]
```

Comment:
Perhaps add the current date? I think it will be useful, especially if we somehow have a request delay

Reply:
Mhm good call